### ISSUE TITLE
Annuler un booking

### ISSUE THREAD

**Issue by Ledoux** [betagouv/pass-culture-browser#279]

**Comment by IonLzr** [comment]
On change juste le label "Réservé" en "Annuler" pour la version MVP : 
https://octo.invisionapp.com/d/#/console/13620708/286234328/preview

Pour la v1, on changera un peu l'aspect des boutons, et donc cette variante également.

**Comment by augnustin** [comment]
Je l'aurais plutôt vu sur la page des réservations car sinon cela signifie qu'on voit l'action "annuler" immédiatement après avoir booké, ce qui me semble un peu pessimiste, mais ça me va, c'est plus simple comme ça. :smile: 

![image](https://user-images.githubusercontent.com/1970156/40605335-5aaa9d9a-6262-11e8-8049-2c8466e2576b.png)

**Comment by IonLzr** [comment]
Je suis d'accord, mais il faut conserver la cohérence graphique : le verso indique soit qu'on peut réserver, soit que c'est déjà fait… quelque soit le contexte. Et voir le bouton "Annuler" immédiatement après la résa a un rôle didactique aussi : je sais où le trouver dès que j'ai fait ma première résa.
 
D'autre part, si l'annulation doit être intégrée qqpart, c'est bien sur la fiche détail – même si j'aurais effectivement préféré lui donner moins de visibilité…

Peut-être qu'on changera si on se rend compte qu'on a trop d'annulations subites :)

**Comment by sofcalca** [comment]
Doublon avec Route d'annulation de booking #124 ?

**Comment by arnoo** [comment]
> Doublon avec Route d'annulation de booking #124 ?

Non, c'est juste que celui-ci est le ticket front. C'est le "[API]" dans le titre qui était faux.
